CHANGELOG.md

```diff
@@ -116,7 +116,12 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.6.0 - TBD
+
+## v2.5.0 - 2022-12
+
+**BREAKING CHANGES**
```

Comment:
Hmmm... I would not say this is breaking.  Those were experimental APIs, subject to change without notice.  Maybe say:

```md
**NOTE**
- feat!: the functions returning bidirectional streams no longer have experimental markers
```

Thoughts?